data/json/items/tool/woodworking.json

```
@@ -77,6 +77,46 @@
     "use_action": [ "CHAINSAW_ON" ],
     "flags": [ "MESSY", "TRADER_AVOID", "NONCONDUCTIVE", "POWERED", "FRAGILE_MELEE" ]
   },
+  {
+    "id": "polesaw_off",
+    "type": "TOOL",
+    "name": { "str": "polesaw (off)", "str_pl": "polesaws (off)" },
+    "description": "This is a lumbering tool intended for trimming hedges and tree branches.  If loaded with gas, using this item will cause it to turn on, making it into an unwieldy polearm.",
+    "weight": "9525 g",
+    "volume": "2500 ml",
+    "longest_side": "275 cm",
+    "price": 9000,
+    "price_postapoc": 450,
+    "to_hit": { "grip": "solid", "length": "long", "surface": "line", "balance": "clumsy" },
+    "bashing": 6,
+    "looks_like": "chainsaw_off",
+    "material": [ "steel", "plastic" ],
+    "symbol": "/",
+    "color": "red",
+    "ammo": [ "gasoline" ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "gasoline": 450 }, "watertight": true, "rigid": true } ],
+    "charges_per_use": 5,
+    "techniques": [ "SWEEP" ],
+    "use_action": [ "CHAINSAW_OFF" ],
+    "flags": [ "NONCONDUCTIVE", "FRAGILE_MELEE", "REACH_ATTACK" ]
```

```suggestion
    "flags": [ "POLEARM", "NONCONDUCTIVE", "FRAGILE_MELEE", "REACH_ATTACK" ]
```
Both versions should have the polearm flag